README.md

```diff
@@ -1,18 +1,18 @@
-1 | Polarization | <ul><li>Feedback process follows traditional methodology, is irregular or non-existant, only being initiated if a large problem is brought forward; feedback and areas of improvement from minorities are either minimalized, avoided, and/or ignored.</ul></li> 
-2 | Minimalization | <ul><li>Use of Produce/Service highlights oppressive policies and law, privacy and security, as well as network infrastructure</ul></li> <ul><li>Semantics and syntax of primary language accommodates mainly majority group, and at most, minimal minority groups</ul></li><ul><li> Consultation for anti-oppressive language is in progress and data analysts are engaged for optimizing clean and race based data</ul></li><ul><li> Process for feedback begins and complex decision making is in development.</ul></li> <ul><li>The working group has minimal representatives of miniority groups</ul></li>
+0 | Denial | <ul><li>The Product/Service project group is homogenous and works in silos.</ul></li><ul><li>Decisions are centred top-down instead of the "Circle Way".</ul></li><ul><li>More time is spent planning and building without a complex feedback system or feedback prioritized.</ul></li><ul><li>Making for an outdated product/service prior to release</ul></li><ul><li>"Firefighting" approach is normalized, compromising long-term (ie. generational) planning development.</ul></li></ul><ul><li>Data is not clean nor secure, demograpic data (race, gender and immigration) isn't relevant in decision making and decisions are based on assimilation and/or capital, leading to hypervisibility or invisibility of marginalized groups.</ul></li><ul><li> Numerous privacy breaches, deemed to harm civil liberities, have been reported.</ul></li></ul></li> <ul><li>Metrics are valued more than customer experience - basic language script is utilized and accessibility is non-existant (automated speech recognition  systems, triggering language, not optimized for Braille, etc.)</ul></li> <ul><li> Heavy documentation is common but record management policies are either ignored or unknown.</ul></li><ul><li> It has not been tested with various minority groups (BIPOC, LGBTQIA2S, Gender Diverse, People with Disabilities, Neuro-Diverse, Residents of Rual Areas).</ul></li>
```

Comment:
Circle Way?  Please reference or juggle the wording.

Reply:
"Making for an outdated product/service prior to release" - sentence fragment

Reply:
"not clean nor" - I hate nor!

Reply:
"Data is not clean"?  Also, chop the sentence down, dropping the parenthesis ().

Reply:
"Numerous privacy breaches, deemed to harm civil liberities, have been reported."  Try "High potential for security breaches."  Tying civil liberties in seems disjointed.

Reply:
Is "Heavy" necessary?  It wouldn't hurt to keep removing parenthesis to make this an easier read.  We can use supplemental information or maybe even footnotes for the most important bits.

Reply:
Corrections completed